examples/MutableMap/MutableMapTrait.dfy

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT
+*******************************************************************************/
+
+// RUN: %verify "%s"
+
+/**
+  *  Specifications that should be satisfied by any implementation of mutable maps.
+  *  Possible instantiations are given in "MutableMapDafny.dfy" (not meant for usage, 
+  *  only exists to verify feasability) and "../../src/MutableMap/MutableMap.dfy" 
+  *  (meant for usage; interfaces with external code, e.g. "../../src/MutableMap/
+  *  MutableMap.java").
+  */
+module {:options "-functionSyntax:4"} MutableMapTrait {
+  trait {:termination false} MutableMapTrait<K(==),V(==)> {
```

Comment:
Unfortunately I think you'd better duplicate this trait in both `src` and `examples`, at least until dafny-lang/dafny#2500 is resolved.